autopush/web/webpush.py

```diff
@@ -63,8 +66,11 @@ def _router_completed(self, response, uaid_data, warning=""):
                 # An empty router_data object indicates that the record should
                 # be deleted. There is no longer valid route information for
                 # this record.
+                self.log.info(format="Dropping User", code=100,
```

Comment:
This code isn't unique. I guess it really doesn't matter because it's in a separate file, but FYI